.github/workflows/add-reviewer-monitoring.yaml

```diff
@@ -0,0 +1,26 @@
+name: Add reviewer on monitoring PRs
+
+on:
+  pull_request:
+    types: [opened]
```

Comment:
Is there a type for when an existing PR is updated to change a file from this list?

Reply:
Yes, I have missed this case, not it should work also on updating existing PR